debian/changelog

```diff
@@ -1,3 +1,15 @@
+git-lfs (0.5.3) stable; urgency=low
```

Comment:
DOH.  I was so focused on changing the date on 0.5.3's entry: e72b187e544087a7e1092aa99cdf1c5aeb0f1ad5